### ISSUE TITLE
support paste disabling in v6

### ISSUE THREAD

**Issue by mymoto23** [tinymce/tinymce#7809]
**Is your feature request related to a problem? Please describe.**

As of [TinyMCE v6](https://www.tiny.cloud/docs/tinymce/6/migration-from-5x/#plugins-paste),

paste plugin is by default activated.

As far as I know, there currently isn't a way to prevent an user from pasting into the editor.

**Describe the solution you'd like**

An option to disable pasting, especially data images, like it was able to in v5.

I remember seeing a `data_image_paste` option which blocked users from pasting images into the editor.

**Describe alternatives you've considered**

The main reason I want to disable this feature is that pasting an image bigger than a certain size (600kb?) doesn't work.

I want to provide my users with a constant behavior when it comes to editing and pasting (images included).

Another solution would be to provide an option to set up a custom handler for image paste events.

This way we (as in developers using TinyMCE) could set up a flow where we upload the image to a CDN and append an image tag with the CDN link.

**Comment by saurabhrai-o** [comment]
I'm also facing the same issue on v6. Do any of you have found any solution for this issue or work around for the mean while ?

**Comment by kilbiller** [comment]
`paste_data_images: false` works fine for me.

**Comment by dev-htf-sa** [comment]
paste_data_images: false option is not working in Firefox

**Comment by Nagendra217** [comment]
Yes, paste_data_images : false ...not working in Firefox.  Also, once image is pasted it is giving the filename as blob or blob1...

**Comment by Mulham** [comment]
same here, i can't prevent the user from pasting images, which will be encoded as base64.
any solution please?

**Comment by KZVMV-EDV** [comment]
Hello,
I found a workaround via "paste" Event handler. The trick: A pasted image in FF contains an <empty String> as pasted data.

```
tinymce.init({
   //...
  setup: function (editor) {
    editor.on("paste",function(e){
            //Prevent Image pasting
            clipboardData = e.clipboardData || window.clipboardData;
            pastedData = clipboardData.getData(\'Text\');
            if(pastedData === ""){
              e.preventDefault();
              return false;
            }
       });
  }
  //...
})
```

**Comment by DeanWunder** [comment]
> Hello, I found a workaround via "paste" Event handler. The trick: A pasted image in FF contains an "empty String" as pasted data.
> 
> ```
> tinymce.init({
>    //...
>   setup: function (editor) {
>     editor.on("paste",function(e){
>             //Prevent Image pasting
>             clipboardData = e.clipboardData || window.clipboardData;
>             pastedData = clipboardData.getData(\'Text\');
>             if(pastedData === ""){
>               e.preventDefault();
>               return false;
>             }
>        });
>   }
>   //...
> })
> ```

Thanks, once I removed the backslashes in `\'Text\'` it worked for me.
